packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -54,22 +54,34 @@ const AccountSignup = ({
         setPWInput(new_password);
     };
 
+    const loggedIn = !!cacheTrackEvents.parseCookies('client_information');
```

Comment:
@agrim-deriv ,can you pass this `loggedIn` property as a core_data.so add this attribute inside the `AnalyticsInitializer` method.And also get the value of `client_information` from cookies.we don't want this parseCookies in the cacheTrackEvents

Reply:
Fixed.